notebooks/04_model_select_and_optimize/README.md

```diff
@@ -34,3 +35,8 @@ To configure this notebook to communicate with your workspace, type in your Azur
     "workspace_name": "<workspace-name>"
 }
 ```
+
+### NNI Configuration
+The NNI command `nnictl` comes installed with the conda environment. 
+In order to use the SMAC tuner, it has to be installed first with the following command 
+`nnictl package install --name=SMAC`.
```

Comment:
Agree on cleaning this up sooner rather than later. Btw, do we want to support docker container image?

Reply:
I added it in `generate_conda.py`, so it is part of the conda environment. I could remove it however; it doesn't need to come with reco_base etc. by default.

Reply:
> Btw, do we want to support docker container image?
yeah that's something that we have discussed also with @msalvaris, I think it would be a nice addition and would make some developments easy

Furthermore, it would give @msalvaris a new incentive to start contributing to this repo :-)

Reply:
https://imgflip.com/i/2x8vpp